docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
<p>Please use "matrix profile" and avoid the use of "MatrixProfile"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Thanks! will change it through out the notebook (will be correct in next commit)